src/main/resources/assets/tconstruct/models/block/table/chest/chest.json

```diff
@@ -1,258 +1,123 @@
+		"2": "tconstruct:block/table/chest/drawer_top",
+		"3": "tconstruct:block/table/chest/drawer_bottom",
+		"particle": "tconstruct:block/table/chest/drawer_front"
+	},
+	"elements": [
+		{
+			"from": [1, 2, 1],
+			"to": [15, 15, 15],
+			"rotation": {"angle": 0, "axis": "y", "origin": [8, 7, 8]},
+			"faces": {
+				"north": {"uv": [1, 0, 15, 13], "texture": "#0"},
+				"east": {"uv": [1, 0, 15, 13], "texture": "#1"},
+				"south": {"uv": [1, 0, 15, 13], "texture": "#1"},
+				"west": {"uv": [1, 0, 15, 13], "texture": "#1"},
+				"up": {"uv": [0, 0, 14, 14], "texture": "#missing"},
+				"down": {"uv": [1, 0, 15, 14], "texture": "#3"}
+			}
+		},
+		{
+			"from": [0, 15, 0],
+			"to": [16, 16, 16],
+			"faces": {
+				"north": {"uv": [0, 0, 16, 1], "texture": "#2"},
+				"east": {"uv": [0, 15, 16, 16], "texture": "#2"},
+				"south": {"uv": [0, 0, 16, 1], "texture": "#2"},
+				"west": {"uv": [0, 15, 16, 16], "texture": "#2"},
+				"up": {"uv": [0, 0, 16, 16], "rotation": 180, "texture": "#2"},
+				"down": {"uv": [0, 0, 16, 16], "texture": "#2"}
+			}
+		},
+		{
+			"from": [1, 0, 1],
+			"to": [3, 2, 3],
+			"rotation": {"angle": 0, "axis": "y", "origin": [9, -5, 9]},
+			"faces": {
+				"north": {"uv": [1, 11, 3, 13], "texture": "#1"},
+				"east": {"uv": [1, 0, 3, 2], "texture": "#1"},
+				"south": {"uv": [13, 0, 15, 2], "texture": "#1"},
+				"west": {"uv": [13, 11, 15, 13], "texture": "#1"},
+				"up": {"uv": [1, 11, 3, 13], "texture": "#1"},
+				"down": {"uv": [1, 11, 3, 13], "texture": "#1"}
+			}
+		},
+		{
+			"from": [1, 0, 13],
+			"to": [3, 2, 15],
+			"rotation": {"angle": 0, "axis": "y", "origin": [9, -5, 21]},
+			"faces": {
+				"north": {"uv": [1, 11, 3, 13], "texture": "#1"},
+				"east": {"uv": [1, 0, 3, 2], "texture": "#1"},
+				"south": {"uv": [13, 0, 15, 2], "texture": "#1"},
+				"west": {"uv": [13, 11, 15, 13], "texture": "#1"},
+				"up": {"uv": [1, 11, 3, 13], "texture": "#1"},
+				"down": {"uv": [1, 11, 3, 13], "texture": "#1"}
+			}
+		},
+		{
+			"from": [13, 0, 1],
+			"to": [15, 2, 3],
+			"rotation": {"angle": 0, "axis": "y", "origin": [21, -5, 9]},
+			"faces": {
+				"north": {"uv": [1, 11, 3, 13], "texture": "#1"},
+				"east": {"uv": [1, 0, 3, 2], "texture": "#1"},
+				"south": {"uv": [13, 0, 15, 2], "texture": "#1"},
+				"west": {"uv": [13, 11, 15, 13], "texture": "#1"},
+				"up": {"uv": [1, 11, 3, 13], "texture": "#1"},
+				"down": {"uv": [1, 11, 3, 13], "texture": "#1"}
+			}
+		},
+		{
+			"from": [13, 0, 13],
+			"to": [15, 2, 15],
+			"rotation": {"angle": 0, "axis": "y", "origin": [21, -5, 21]},
+			"faces": {
+				"north": {"uv": [1, 11, 3, 13], "texture": "#1"},
+				"east": {"uv": [1, 0, 3, 2], "texture": "#1"},
+				"south": {"uv": [13, 0, 15, 2], "texture": "#1"},
+				"west": {"uv": [13, 11, 15, 13], "texture": "#1"},
+				"up": {"uv": [1, 11, 3, 13], "texture": "#1"},
+				"down": {"uv": [1, 11, 3, 13], "texture": "#1"}
+			}
+		}
+	],
+	"display": {
```

Comment:
Many of these display properties are redundant to `block/block`. It is not necessary to include every display property, only the ones you changed.